src/ProfilerEvent.php

```diff
@@ -36,7 +36,7 @@ class ProfilerEvent extends Event
      *
      * @return ApplicationInterface|null
      */
-    public function getApplication()
+    public function getApplication(): ?ApplicationInterface
```

Comment:
Please do not add native php typehints in this PR as this introduces a BC break. 
Focus on PHP 8.0 compatibility only.
Thanks!